app/src/main/java/com/example/awesomelanguagelearning/login_signup/ui/Controls.kt

```diff
@@ -0,0 +1,126 @@
+package com.example.awesomelanguagelearning.login_signup.ui
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.example.awesomelanguagelearning.R
+import com.example.awesomelanguagelearning.core.ui.theme.AppTheme
+import com.example.awesomelanguagelearning.core.ui.views.HorizontalSpacer
+import com.example.awesomelanguagelearning.core.ui.views.ImageButton
+import com.example.awesomelanguagelearning.core.ui.views.TextButton
+import com.example.awesomelanguagelearning.core.ui.views.TextTitle
+import com.example.awesomelanguagelearning.core.ui.views.TextWithClickablePart
+
+@Composable
+fun Controls(
```

Comment:
@igel911 I think that this component is too big and has a lot of responsibilities, hard to reuse and I think it's better to split it into smaller components

Reply:
@slothdeboss done